knowledge-base/kb-datetimepicker-scrollviewer-clock-items.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Show ScrollViewer for the time display items
```

Comment:
Should we place the name of the control - DateTimePicker somewhere in the titles/description? Such as in the kb article for the RadRibbonView.